hhpc.c

```diff
@@ -226,9 +227,10 @@ static void waitForMotion(Display *dpy, Window win, int timeout) {
 static int parseOptions(int argc, char *argv[]) {
     int option = 0;
 
-    while ((option = getopt(argc, argv, "i:v")) != -1) {
+    while ((option = getopt(argc, argv, "i:hv")) != -1) {
         switch (option) {
             case 'i': gIdleTimeout = atoi(optarg); break;
+            case 'h': gDisplayHelp = 1; break;
             case 'v': gVerbose = 1; break;
             default: return 0;
```

Comment:
Maybe this case (which encompasses '?') can be used to return 1? I don't remember exactly why I made this return 0.

Reply:
Never mind, I just had the logic inverted, thinking in terms of UNIX style (return 0 is success). But here it's inverted. Probably I should've used `stdbool` here. Can't remember why I didn't do that here either.